compiler/modules/types/common.bal

```diff
@@ -135,10 +135,8 @@ function bddPaths(Bdd b, BddPath[] paths, BddPath accum) {
     }
 }
 
-// JBUG (33706) should be able to use clone
-// what happens is after clone, empty pos and neg, which were == but not ===, become ===
 function bddPathClone(BddPath path) returns BddPath {
```

Comment:
Get rid of this function. Just inline calls to `clone`.

Reply:
Fixed with https://github.com/ballerina-platform/nballerina/pull/847/commits/bfb334cc593c040c55fb01d7f785b933b1a984cf